build_tools/docker/manage_images.py

```diff
@@ -64,7 +64,7 @@
     'cmake-python-swiftshader': ['cmake-python-vulkan', 'swiftshader'],
     'cmake-python-vulkan': ['cmake-python', 'vulkan'],
     'rbe-toolchain': ['vulkan'],
-    'swiftshader': ['cmake'],
+    'swiftshader': ['cmake', 'vulkan'],
```

Comment:
You did not add this dependency. The swiftshader image does not depend on the vulkan image. The swiftshader image is just the one called "swiftshader" that does the swiftshader install build that is copied by other images.

Reply:
done